README.md

```diff
@@ -65,8 +65,7 @@ instance HasDefinitions MyContracts where
 3. Write your main entrypoint for the application and the configuration file
 
 ```haskell
-import BotPlutusInterface qualified
-import BotPlutusInterface.Config qualified as BotPlutusInterface
+d601900dab2e364aea1084bcb8281116e4724309
```

Comment:
what is this?

Reply:
probably some random hash, not sure how it got there. But Fixed it.